src/selectedtests/task_mappings/cli.py

```diff
@@ -107,14 +113,19 @@ def create(
             raise click.ClickException(
                 "A module source file regex is required when a module is being analyzed"
             )
-            return
         else:
             module_file_regex = re.compile(module_source_file_regex)
 
     LOGGER.info(f"Creating task mappings for {evergreen_project}")
 
     mappings = TaskMappings.create_task_mappings(
-        evg_api, evergreen_project, start_date, end_date, file_regex, module_name, module_file_regex
+        evg_api,
+        evergreen_project,
+        start_date,
+        end_date,
+        file_regex,
+        module_name,
+        module_file_regex,
```

Comment:
Should `build_variant_regex` be passed to this call?

Reply:
Whoops, forgot to commit those changes. Good catch, fixed